virttest/libvirt_xml/base.py

```diff
@@ -320,6 +320,70 @@ def setup_attrs(self, **attrs):
                 target_obj.setup_attrs(**value)
                 setattr(self, key, target_obj)
 
+    def fetch_attrs(self):
```

Comment:
Could you elaborate a little bit more about what issue this method want to address?
I assume we previously have some common sense that regarding xml management, we firstly get mapped object regarding relevant xml, then we can manipulate the object to get attrs or others